nncf/api/composite_compression.py

```diff
@@ -46,6 +46,26 @@ def add(self, child_loss: CompressionLoss) -> None:
         """
         self._child_losses.append(child_loss)
 
+    def load_state(self, states: List[Dict[str, object]]) -> None:
+        """
+        Loads the composite compression loss state.
+
+        :param states: Output of `get_state()` method.
+        """
+        for child_loss, child_state in zip(self._child_losses, states):
+            child_loss.load_state(child_state)
```

Comment:
Are you sure that `states` order are correspond to the order of `self._child_losses` ?

Reply:
I would like to see here matching not by order but matching by name

Reply:
the same comment from my side. It's useful for algo mixing when number of loaded algorithms don't match to the number of algorithms in state

Reply:
The way we handle multiple compression and multiple loss state are correspond to the way we handle multiple scheduler state now. I think to cover this issue we should open enhancement ticket